tests/electron/modules/calendar_spec.js

```diff
@@ -12,10 +12,11 @@ describe("Calendar module", () => {
 		await expect(elem.isVisible()).resolves.toBe(true);
 		return true;
 	};
+	//doTestTableContent(".testNotification tr", "td.elementCount", "12", first)).resolves.toBe(true);
```

Comment:
can this be removed?

Reply:
yes,will do, copied to verify the call parms matched  

there was a day where it worked and then suddenly didn't. i had the test module in defaults (had to change other files to allow that., wrong)

then moved module to tests and nothing worked
cause of the path problem in app.js, cause of foreign setting

Reply:
yes